data_generators/xcspec_extractor.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
```

Comment:
What if python3 is not installed? I think we can only count on python 2 by default no?

Reply:
Given this is only run manually, I'm not too too concerned -- we check in it's artifact, and the rules don't use the generator directly

Reply:
Sure, but it doesn't look like you're using anything from python 3 in particular... shouldn't be too hard to keep it python2/python3 compatible? I agree that i's not a biggie though...